tagstudio/src/qt/main_window.py

```diff
@@ -36,7 +36,7 @@ class Ui_MainWindow(QMainWindow):
   
     def __init__(self, driver: "QtDriver", parent=None) -> None:
```

Comment:
```suggestion
    def __init__(self, driver: QtDriver, parent=None) -> None:
```

Reply:
this is not a line I have touched, so I consider that out of scope.